sklearn/linear_model/least_angle.py

```diff
@@ -757,7 +757,7 @@ def _lars_path_solver(X, y, Xy=None, Gram=None, n_samples=None, max_iter=500,
 ###############################################################################
 # Estimator classes
 
-class Lars(LinearModel, RegressorMixin, MultiOutputMixin):
+class Lars(LinearModel, RegressorMixin):
```

Comment:
Is there a way to keep the mixin here, but to override the tag in subclasses that do not handle multioutput ?

Reply:
I now solved it with intermediate classes so we now have `_baseLars` without `MultiOutputMixin` and `Lars` class that inherits `_baseLars` and has `MultiOutputMixin`.

The way tags are implemented does not allow for mixins that negate other mixins. Probably it wouldn't be a good idea anyway.